### PR TITLE
fix $(RATING_ICONS) variable substitution

### PR DESCRIPTION
**Pull request by ralfbrown**
As noted in a comment to #11019, a rejected image currently shows zero stars.  This patch makes $(RATING_ICONS) expand to the indended X for rejected images.

I left in the old check for stars==6 meaning rejected, as presumably some very old sidecars may have that setting.

### CODE REVIEW

**Comment by phweyland** [comment]
Could you please add $(RATING ICONS) in the list of variables in gtkentry.c

Would the fix of $(RATING ICONS) apply on $(STARS) as well (for rejected images) ?

**Comment by ralfbrown** [comment]
Yes, the fix applies to $(STARS) as well, since $(RATING_ICONS) simply takes the number that $(STARS) returns and converts it to the icon string.  So a rejected image will have $(STARS) show -1, where before it would show the pre-rejection star rating for images that are using the separate rejected flag instead of the 6-stars=rejected convention.

**Comment by TurboGit** [comment]
I have the notification on my mail box, but when I click on the link it goes nowhere.

A GitHub bug? I would just do some minor change (formatting for example) and push again.

**Comment by phweyland** [comment]
> but it isn't showing up here.... (

It's visible now :)

**Comment by ralfbrown** [comment]
Must have been a server-side glitch.

File: src/gui/gtkentry.c [hunk 1]
```diff
@@ -210,19 +211,22 @@ const dt_gtkentry_completion_spec *dt_gtkentry_get_default_path_compl_list()
           { "ELEVATION", N_("$(ELEVATION) - elevation") },
           { "STARS", N_("$(STARS) - star rating as number (-1 for rejected)") },
           { "RATING_ICONS", N_("$(RATING_ICONS) - star/reject rating in icon form") },
-          { "LABELS", N_("$(LABELS) - colorlabels") },
+          { "LABELS", N_("$(LABELS) - color labels at text") },
```
Comment (phweyland) [inline src/gui/gtkentry.c:214]: as text ?

Comment (TurboGit) [reply src/gui/gtkentry.c:214]: Cannot test right now but yes, IIRC the output is using color names, for example `red, green`.

Comment (phweyland) [reply src/gui/gtkentry.c:214]: I meant "as" instead of "at" ... :)

Comment (TurboGit) [reply src/gui/gtkentry.c]: And you know what I did read "as" :)

**Review by TurboGit — APPROVED** [review]
Thanks!

**Comment by TurboGit** [comment]
Merged into master and darktable-3.8.x branch.

**Pull request merged**